File: SeleniumJava/com.crm.seleniumM4/src/webpageVerification/VerifyByURL.java
package webpageVerification;

import org.openqa.selenium.chrome.ChromeDriver;

public class VerifyByURL {

	public static void main(String[] args) throws InterruptedException {
		//Open the browser. 
		String expected_result = "https://demowebshop.tricentis.com/";
		ChromeDriver driver = new ChromeDriver();
		//maximize the browser. 
		driver.manage().window().maximize(); 
		// Waiting condition 
		Thread.sleep(3000);
		// enter into dws. 
		driver.get(expected_result);
		String actual_result = driver.getCurrentUrl(); 
		if(expected_result.equals(actual_result))
			System.out.println("I'm in dws page");
		else
			System.out.println("I'm not in dws page");	
		
		driver.close();
	}

}

File: SeleniumJava/com.crm.seleniumM4/src/webpageVerification/VerifyByURL.py
from selenium import webdriver
import time; 

driver = webdriver.Chrome() 
driver.maximize_window()
expected_result = "https://demowebshop.tricentis.com/";
driver.get(expected_result)
actual_result = driver.current_url
if(expected_result == actual_result):
    print("I am in DWS page")
else:
    print("I am not in dws page")
